sdks/js/packages/core/react/components/organization/billing/invoices/index.tsx

```diff
@@ -0,0 +1,174 @@
+import { DataTable, EmptyState, Flex, Link, Text } from '@raystack/apsara';
+import { ColumnDef } from '@tanstack/react-table';
+import dayjs from 'dayjs';
+import { useCallback, useEffect, useMemo, useState } from 'react';
+import Skeleton from 'react-loading-skeleton';
+import { useFrontier } from '~/react/contexts/FrontierContext';
+import { V1Beta1Invoice } from '~/src';
+import { capitalize } from '~/utils';
+
+interface InvoicesProps {
+  organizationId: string;
+  billingId: string;
+  isLoading: boolean;
+}
+
+interface getColumnsOptions {
+  isLoading?: boolean;
+}
+
+export const getColumns: (
+  options: getColumnsOptions
+) => ColumnDef<V1Beta1Invoice, any>[] = ({ isLoading }) => [
+  {
+    header: 'Date',
+    accessorKey: 'effective_at',
+    meta: {
+      style: {
+        paddingLeft: 0
+      }
+    },
+    cell: isLoading
+      ? () => <Skeleton />
+      : ({ row, getValue }) => {
+          return (
+            <Flex direction="column">
+              <Text>{dayjs(getValue()).format('DD MMM YYYY')}</Text>
+            </Flex>
+          );
+        }
+  },
+  {
+    header: 'Status',
+    accessorKey: 'state',
+    meta: {
+      style: {
+        paddingLeft: 0
+      }
+    },
+    cell: isLoading
+      ? () => <Skeleton />
+      : ({ row, getValue }) => {
+          return (
+            <Flex direction="column">
+              <Text>{capitalize(getValue())}</Text>
+            </Flex>
+          );
+        }
+  },
+  {
+    header: 'Amount',
+    accessorKey: 'amount',
+    meta: {
+      style: {
+        paddingLeft: 0
+      }
+    },
+    cell: isLoading
```

Comment:
Its repeating many time 
Should we create a `WithSkeleton` component with loading and component props?

Reply:
sure, lets do it in apsara